lib/libc/stdio/lib_sscanf.c

```diff
@@ -406,7 +407,7 @@ int vsscanf(FAR const char *buf, FAR const char *fmt, va_list ap)
 					for (tc = fmt; isdigit(*fmt); fmt++);
 					strncpy(tmp, tc, fmt - tc);
 					tmp[fmt - tc] = '\0';
-					width = MIN(sizeof(tmp) - 1, atoi(tmp));
+					width = atoi(tmp);
```

Comment:
Since length of tmp array is quite large, I think its better to use atol ot atoll in order to prevent overflow.